build/commands/scripts/commands.js

```diff
@@ -3,6 +3,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// Check environment before doing anything.
+require('../lib/checkEnvironment')
```

Comment:
Just confirming this executes before iOS builds too?

Reply:
`sync`/`build` commands are executed as part of iOS builds.